### ISSUE TITLE
Create a visualization view in WebVR (and WebAR?) using A-Frame

### ISSUE THREAD

**Issue by cvan** [mozilla/lightbeam-we#202]
Hi, my name is Chris Van Wiemeersch (@cvan). I work on the [WebVR team](https://vr.mozilla.org/) here at Mozilla. I was one of the co-creators of [A-Frame](https://aframe.io/), the framework for easily creating WebVR experiences.

@potch pointed me to this WebExtension port of Lightbeam. I'm really excited that this exists. Really nice work!

@caseyyee and I would love to help out in creating a WebVR view for the visualization in the Lightbeam extension. Let us know if you have any specific ideas. If it's all right with y'all, we can discuss in this issue and open Pull Requests. Let me know otherwise.

I'm available on the MoCo Slack, but I'd definitely prefer to keep conversations public and transparent on GitHub. If a 3D, WebVR/AR visualization is out of scope for Lightbeam, I understand. I figured I'd file an issue to kickstart the conversation. Thanks!

**Comment by princiya** [comment]
Hi @cvan, thanks for reaching out :) Here is my Mozfest proposal - https://github.com/MozillaFoundation/mozfest-program-2017/issues/319 and here is my - [initial POC](http://faithful-hope.glitch.me/). 

I tried to play around a bit with A-Frame, to come up with this POC. Let me know what you think about this and I would certainly need your help going forward. Thank you.

**Comment by caseyyee** [comment]
Amazing :)  Love that you already have a start on it.  Happy to jump in and help get this going.   This is the sort of project that will really help elevate WebVR.   Super easy to basically graft VR onto any existing web apps.

**Comment by jonathanKingston** [comment]
Super excited you both found this :)

**Comment by cvan** [comment]
@princiya: awesome work! is there a GitHub repo we can work out of?

**Comment by princiya** [comment]
@cvan @caseyyee here is the repo [https://github.com/princiya/lightbeam-aframe](https://github.com/princiya/lightbeam-aframe). This is a work in progress and inputs, feedback, suggestions from all possible angles is highly appreciable.

- As part of POC, I created `data.json` to serve static nodes and links which represent the trackers.
- The next step is to create a dynamic view of the first party and third party trackers.

**Comment by jonathanKingston** [comment]
@princiya it might make sense to create a branch here and use the dynamic data we have here?

**Comment by princiya** [comment]
@jonathanKingston I was not sure to add it here, do you think it makes more sense to do so?

**Comment by jonathanKingston** [comment]
@princiya to make a branch of the lightbeam code with web vr enabled using the storage code rather than a dump of the data.

**Comment by cvan** [comment]
@princiya awesome! I can help with making a PR of that branch of Lightbeam use the live code from the WebExtensions APIs. Would you like me to cherry-pick your commits, or do you want to create the branch?

**Comment by caseyyee** [comment]
If I can get a good dataset dump would be helpful to start prototyping some visuals if you have it :)

**Comment by jonathanKingston** [comment]
@caseyyee I would rather not dump mine, however it's super easy to get. Install Lightbeam from AMO then click the save button after browsing a few news sites. :)

I know @princiya is travelling at the moment but I can help with any questions you might have etc.

**Comment by cvan** [comment]
@caseyyee: yeah just use Dev Edition or Nightly. and clone this repo (and the other one), and enable loading of the extension from `about:debugging`.
